src/Input/Silk.NET.Input.Glfw/ReadOnlyListEnumerator.cs

```diff
@@ -17,13 +17,13 @@ public ReadOnlyListEnumerator(IReadOnlyList<T> list)
         {
             _list = list;
             _i = 0;
-            Current = default;
+            Current = default!;
```

Comment:
Why is this maintaining a `Current` in the first place? Why not make `Current` just use `_i` like `Current => _list.Items[_i]`